lib/simple_form/components/errors.rb

```diff
@@ -34,6 +34,14 @@ def full_error_text
         has_custom_error? ? options[:error] : full_errors.send(error_method)
       end
 
+      def form_with_object_has_errors?
```

Comment:
We could name this method of `model_with_errors?`, what do you think?

Reply:
Yes I like more that name, but what you think to call it `object_with_errors?` (just to respect the same variable name that we use in the method)

Reply:
Done, I have amended my commit and renamed the method.